### DISCUSSION TITLE
How do you change archive page size per archive?

### DISCUSSION THREAD

**Discussion by selaromdotnet** [PiranhaCMS/piranha.core#2075] [Q&A]
Hi, the config page in manager lets you set the global archive page size, but it says you can also configure this per archive type. However, I can't find any information on how to do this, can someone point me in the right direction? thanks!

**Comment by tidyui** [comment]
This is basically us saying "This is where you specify your desired archive page size, but we can't guarantee that the developer of the site cares about this setting", but in a more formal way :) 

Since it's up to the developer to fetch the archive, you can also choose to do it differently depending on context and content type, but there's no UI setting for it. If the optional parameter `pageSize` is omitted from `IArchiveService.GetByIdAsync` the configured page size will be used.

https://github.com/PiranhaCMS/piranha.core/blob/master/core/Piranha/Services/IArchiveService.cs#L45

**Comment by selaromdotnet** [comment]
ahhh gotcha, thanks for clearing that up!
